.gitignore

```diff
@@ -22,3 +22,6 @@ _testmain.go
 *.exe
 *.test
 *.prof
+
+zaproxy/
+zap-extensions/
```

Comment:
These should be removed, the repos should live at the same level as the `zap-api-go` not under it.

Reply:
What if we link these two projects as submodules instead? This way, users won't need to clone them manually into the correct location. I added them to the .`gitignore` file to ensure they are not committed accidentally.

Reply:
The repos are already cloned, per dev guide.